src/templates/token.tsx

```diff
@@ -87,14 +109,32 @@ const css = `
     word-break: break-all;
   }
 
+  .link-text {
+    text-decoration: underline !important;
+    color: ${colors.primary} !important;
+  }
+  .link-text:hover {
+    color: ${colors.tertiary} !important;
+  }
   .hover-img-button:hover {
     opacity: 0.7;
   }
 `
 
+interface TokenProps {
+  secret: string
+  domain: string
+}
+
 const assetsEndpoint = `${env.SEALCRED_ADDRESS}/img/email`
 
-const generateTokenPage = ({ secret }: { secret: string }) => {
+const generateTokenPage = ({ secret, domain }: TokenProps) => {
```

Comment:
Maybe instead of `secret` we should have `token` to better understand which secret?